src/test/regress/expected/ensure_no_intermediate_data_leak_0.out

```diff
@@ -0,0 +1,19 @@
+------
+-- THIS TEST SHOULD IDEALLY BE EXECUTED AT THE END OF
+-- THE REGRESSION TEST SUITE TO MAKE SURE THAT WE
+-- CLEAR ALL INTERMEDIATE RESULTS ON BOTH THE COORDINATOR
+-- AND ON THE WORKERS. HOWEVER, WE HAVE SOME ISSUES AROUND
+-- WINDOWS SUPPORT SO WE DISABLE THIS TEST ON WINDOWS
+------
+SELECT pg_ls_dir('base/pgsql_job_cache') WHERE citus_version() NOT ILIKE '%windows%';
+ pg_ls_dir 
+-----------
+(0 rows)
+
+SELECT run_command_on_workers($$SELECT pg_ls_dir('base/pgsql_job_cache') WHERE citus_version() NOT ILIKE '%windows%'$$);
+ run_command_on_workers 
+------------------------
+ (localhost,9060,t,"")
```

Comment:
We could update this test to only select cases when there are directories, so that we only have 1 expected result, as opposed to 3